ChangeLog

```diff
@@ -1,6 +1,9 @@
 CHANGES
 =======
 
+FEATURE: Ldap support for user management
+
+1.7.0
```

Comment:
I understand that the plan is to close release 1.7.0 before adding the LDAP support.

If that is correct, I'd sugges to do close the 1.7.0 release in an independent PR (mirroring this one for instance PR https://github.com/telefonicaid/orchestrator/pull/100)

Reply:
Version1.7.0 closed in PR https://github.com/telefonicaid/orchestrator/pull/118

NTC